ballerina/modules/wssec/ws_security_methods.bal

```diff
@@ -163,9 +178,12 @@ public isolated function applySymmetricBinding(xml envelope, *SymmetricBindingCo
 # Apply asymmetric binding security policy with X509 token to the SOAP envelope.
 #
 # + envelope - The SOAP envelope
+# + soap12 - A boolean flag. Set to `true` for SOAP 1.2, or `false` for SOAP 1.1.
 # + asymmetricBinding - The `AsymmetricBindingConfig` record with the required parameters
 # + return - A `xml` type of SOAP envelope if the security binding is successfully added or else `wssec:Error`
-public isolated function applyAsymmetricBinding(xml envelope, *AsymmetricBindingConfig asymmetricBinding) returns xml|crypto:Error|Error {
+public isolated function applyAsymmetricBinding(xml envelope, boolean soap12, 
```

Comment:
since these are breaking changes, we have to bump a minor version right?

Reply:
Fixed by [cc94acb](https://github.com/ballerina-platform/module-ballerina-soap/pull/112/commits/cc94acbf9b4e207de78305183521d5ad05c4d40a)